src/main/java/org/micks/champmaker/register/RegisterService.java

```diff
@@ -36,13 +36,13 @@ public List<Long> getRegisteredPlayers(long champId, long teamId) {
     }
 
     public List<Long> getRegisteredTeams(long champId) {
-        List<RegisterEntity> registerEntities = registerRepository.findByChampId(champId);
+        List<RegisterTeamEntity> registerEntities = registerRepository.findByChampId(champId);
         return registerEntities.stream()
-                .map(RegisterEntity::getTeamId).collect(toList());
+                .map(RegisterTeamEntity::getTeamId).collect(toList());
     }
 
     public void registerPlayer(long champId, RegisterPlayerDTO registerPlayerDTO) {
-        RegisterPlayerEntity registerPlayerEntity = new RegisterPlayerEntity(champId, registerPlayerDTO.getPlayerId());
+        RegisterPlayerEntity registerPlayerEntity = new RegisterPlayerEntity(champId, registerPlayerDTO.getMealId(), registerPlayerDTO.getPlayerId());
```

Comment:
Wrong order of paremeters - in RegisterPlayerEntity constructor mealId is first parameter and here you put it as second parameter. This should be also checked in tests with Postman if proper ids are added to database.